routing_common/pedestrian_model.cpp

```diff
@@ -122,6 +122,16 @@ VehicleModel::LimitsInitList YesBridleway(VehicleModel::LimitsInitList res = kDe
   return res;
 }
 
+/// @see Turkey_UsePrimary, Georgia_UsePrimary test.
```

Comment:
Why only these countries? Can you please explain in the comments?

Reply:
Countries where we had claims already.
I suppose, it is common for _mountain_ regions where "primary" is the one and only road between towns. In a _megapolis_ primary road is usually not so suitable for pedestrians (if opposite is not set)